src/poliastro/threebody/cr3bp_char_quant.py

```diff
@@ -0,0 +1,236 @@
+"""
+@author: Dhruv Jain, Multi-Body Dynamics Research Group, Purdue University
+
+Useful ideas:
+1. To non-dimensioanlize a physical position vector: divide by l*
+2. To dimensioanlize a [nd] position vector: multiply by l*
+3. To non-dimensionalize a physical veloctiy vecotr, multiply by t*/l*
+4. To dimensionalize a [nd] veloctiy vecotr, multiply by l*/t*
+5. Position of P1 = [-mu, 0, 0] [nd] {x,y,z}
+5. Position of P2 = [1-mu, 0, 0] [nd] {x,y,z}
+"""
+from astropy import units as u
+
+import poliastro.constants.general as general
+import poliastro.constants.mean_elements as dist
+
+
+class SystemChars:
+    """Computes and stores the properties (mu, l*, t*) of a CR3BP system
+    'mu': mass ratio of P1-P2 primary bodies
+    'l*': characterisitic lenght of P1-P2 primary bodies
+    't*': characterisitic time of P1-P2 primary bodies
+    If P2 is more massive than P1 then swap the Pi, so that P1 is the more massive body
+    """
+
+    def __init__(self, p1="Earth", p2="Moon"):
+        """
+        Constructor
+
+        Parameters
+        ----------
+        p1 : string
+           'body_i'. The deafult is Earth
+        p2 : string
+           'body_j'. The deafult is Moon
+        """
+
+        mu, lstar, tstar, p1, p2 = self.BodiesCharCompute(
+            p1.lower(), p2.lower()
+        )  # lower() to ensure lowercase characters
+        units_nd = u.def_unit(
+            "[nd]"
+        )  # Define custome units, [nd]: non-dimensional unit
```

Comment:
Do we need a custom non-dimensional unit though? We have `u.one` for that (aka `u.dimensionless_unscaled`, although [I introduced the `u.one` shorthand in Astropy 8 years ago](https://github.com/astropy/astropy/pull/1980))

Reply:
We will need to define our own [nd] units. But after reading this piece of code again, I realized that mu is dimensionless, so it will make sense to use u.one.

The idea will be to non-dimensionalize position by l*, so 1 [nd] = l* km. Example: Earth-Moon -> 1 [nd] position = 384400 km. 

We can get rid of this specific unit but I will add a "[nd]" for position and "[nd]" for velocity as the [nd] quantities are still dimensional quantities. They are just scaled by the characteristic quantities which is why the units are called non-dimensional and not dimensionless.

Reply:
I see. What about something more explicit? This is what I did in an experiment to introduce canonical units everywhere in poliastro https://github.com/poliastro/poliastro/pull/708/:

```python
du = u.def_unit("du", norm(r))
tu = u.def_unit("tu", np.sqrt((1 * du) ** 3 / source_body.k))

r_value = r.to_value(du)
v_value = v.to_value(du / tu)
```

(just an idea)